docs/assets/templates/tools/template-maximum.dockstore.yml

```diff
@@ -0,0 +1,61 @@
+# The first line refers to the version 1.2 of the .dockstore.yml schema
```

Comment:
I liked calling this `template-best-practice` rather than `maximum`. It signals the value of doing all of this.